src/main/java/nl/dtls/fairdatapoint/api/controller/MetadataController.java

```diff
@@ -530,49 +491,52 @@ public String storeDistribution(final HttpServletRequest request,
     private String getRequesedURL(HttpServletRequest request) {
         String url = request.getRequestURL().toString();
         LOGGER.info("Original requesed url " + url);
+        
         if (url.endsWith("/")) {
             url = url.substring(0, url.length() - 1);
-        }      
-        List<String> rdfExt =  RDFWriterRegistry.getInstance()
-                .getKeys().stream()
-                    .map(RDFFormat::getDefaultFileExtension)
-                    .collect(Collectors.toList());
+        }
         
+        List<String> rdfExt = RDFWriterRegistry.getInstance().getKeys().stream()
+                .map(RDFFormat::getDefaultFileExtension).collect(Collectors.toList());
```

Comment:
I think the chained list with each new step on a new line is more readable, which formatter are you using to format this?

Reply:
I am using netbean's default formatter for java.  Do u want me to change the above line like one below?

RDFWriterRegistry.getInstance()
.getKeys()
.stream()
.map(RDFFormat::getDefaultFileExtension)
.collect(Collectors.toList());

Reply:
To me that's more readable, but that might be just me. Google's style guide has no specific mention of method chaining, but the closest point they make is [one statement per line](https://google.github.io/styleguide/javaguide.html#s4.3-one-statement-per-line).

Reply:
I will change it anyway